src/api/resources/openApi.json

```diff
@@ -475,7 +475,7 @@
           "type": {
             "type": "string",
             "description": "Source type of map data.",
-            "enum": ["tilejson", "wms", "wmts"],
+            "enum": ["tilejson", "wms", "wmts", "mapstyleJSON", "s-57"],
```

Comment:
What is `mapstyleJSON`?

Should S-57 be with capital S?

Reply:
MapstyleJSON refers to mapbox style files that describe vector map sources and styles.

So you can have a vector map and apply different styles to it e.g. day / night.

Reply:
Updated S-57 to uppercase (also aligned WMS & WMTS by making them upper case).
Changes `mapstyleJSON` to `mapboxstyle`